src/EncompassRest/Loans/RateLocks/LockStatus.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Runtime.Serialization;
+
+namespace EncompassRest.Loans.RateLocks
+{
+    /// <summary>
+    /// LockStatus
+    /// </summary>
+    public enum LockStatus
+    {
+        /// <summary>
+        /// Requested
+        /// </summary>
+        [EnumMember(Value = "Requested")]
```

Comment:
If the name matches the actual value you don't need to specify an `EnumMemberAttribute`.

Reply:
I did that for consistency. Do you prefer I remove them?

Reply:
I elide them in that case but you can leave it.